packages/vscode-extension/package.nls.json

```diff
@@ -92,6 +92,8 @@
     "teamstoolkit.commands.validateManifest.title": "Validate Manifest File",
     "teamstoolkit.commands.viewSamples.title": "View Samples",
     "teamstoolkit.commands.zipPackage.title": "Zip Teams App Package",
+    "teamstoolkit.commmands.addWebpart.title": "Add SPFx web part",
```

Comment:
This should be capitalized, "Add SPFx Web Part"

Reply:
Updated